apps/backend/src/main.ts

```diff
@@ -1,18 +1,49 @@
-/**
- * This is not a production server yet!
- * This is only a minimal backend to get started.
- */
-
+import cors from 'cors';
 import express from 'express';
+import { Connection } from 'typeorm';
+
+import { Menu } from './data/models/menu';
+import { ModifierCategory } from './data/models/modifierCategory';
+import { Modifiers } from './data/models/modifiers';
+import { Product } from './data/models/product';
+import { Store } from './data/models/store';
+import storeRoutes from './routes/store';
 
 const app = express();
+const port = process.env.port || 3333;
+
+const corsOptions = {
+  origin: 'http://localhost:4200'
+};
+
+export const connection = new Connection({
```

Comment:
aren't we reading it from ormconfig? saw the same block in Nina's code

Reply:
yes, you're right. Initially we tried to import this config in main.ts file. But there was a problem with importing module from ormconfig.js file. As result, we decided to make a connection with the service like it was done in documentation, by creating a constant and all the data about connection to database